docker/Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+# To build this container, go to ESMValCore root folder and execute :
+# docker build -t ${TAG_OF_YOUR_CHOICE} . -f docker/Dockerfile
+FROM continuumio/miniconda3
+
+COPY . /src/ESMValTool
+WORKDIR /src/ESMValTool
```

Comment:
It would be good to remove the `/src` dir again after installation is complete, to reduce the size

Reply:
done

Reply:
No longer done, as in order to make it work with the .dockerignore I had to use the -e option (no clue about why). The good news is that we are greatly reducing the used space